src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -329,6 +329,7 @@ namespace NServiceBus
     public class static EndpointDiagnosticSettingsExtensions
     {
         public static void AddStartupDiagnosticsSection(this NServiceBus.Settings.ReadOnlySettings settings, string sectionName, object section) { }
+        public static void AddStartupDiagnosticsSection(this NServiceBus.Features.FeatureConfigurationContext context, string sectionName, object section) { }
```

Comment:
why adding this new API?

Reply:
I felt like a better way to expose this to features since all of them should really expose diagnostics?